### PR TITLE
Etiology

### PR DESCRIPTION
**Pull request by lkorczowski**
- [X] read etiology
- [x] merge with data_info in NB
- [x] visualize data

### CODE REVIEW

**Comment by lkorczowski** [comment]
@RobinGuillard 
Unfortunately, it seems that the current NB don't have the modifications made during the PR #101 
I supposed that the new NB have been overwritten at some point so we lost a bit.

Tomorrow, I'll take back the NB at the state https://github.com/lkorczowski/Tinnitus-n-Sleep/blob/46c1dc52eea88b0b99243d1e7237dd885dcf8058/notebooks/Middle_ear_Inter_subject_analysis.ipynb
and do the modifications again.

**Comment by lkorczowski** [comment]
@RobinGuillard 
note that the difference between NB, ie
```
# NB Bruxism
var[59]="control" #1PA31
var[61]="control" #1CB23
var[70]="control" #3CL02
```

```
# NB MEMA
var[70] = "control" #3CL02
var[71] = "control" #3TS03
var[61] = "control" #1CB23
#var[65] = "control" #3BS04
var[67] = "control" #3KB06
var[59] = "control" #1PA31
#var[40] = "control" #1UC22
```
is not a good pratice. Please update data_info instead ^^

**Comment by lkorczowski** [comment]
Review NB in reviewNB https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/111/

File: tinnsleep/visualization.py [hunk 1]
```diff
@@ -328,3 +334,128 @@ def zoom_effect(ax1, ax2, xmin=None, xmax=None, prop_lines={}, **kwargs):
...
+
+
+def etiology_report_with_plot(data, etiology, variable, hue=None, hue_stats=None, verbose=2):
+    """
+    Parameters
+    ----------
+
+    Returns
+    -------
+    """
```
Comment (RobinGuillard) [inline tinnsleep/visualization.py]: Complete description here

File: tinnsleep/visualization.py [hunk 2]
```diff
@@ -328,3 +334,128 @@ def zoom_effect(ax1, ax2, xmin=None, xmax=None, prop_lines={}, **kwargs):
...
+    categories = list(data[etiology].unique())
```
Comment (RobinGuillard) [inline tinnsleep/visualization.py:464]: Add lines:
'''
print('')
print(etiology)
'''
for better clarity in NB

**Review by RobinGuillard — APPROVED** [review]

**Pull request merged**